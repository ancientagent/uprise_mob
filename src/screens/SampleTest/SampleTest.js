import React from 'react';
import { View, Text } from 'react-native';
import { Button } from 'react-native-elements';
import styles from './SampleTest.styles';
// TODO this is for testing purpose we added code here. this will no longer be here
import viewProfileImage from '../../../assets/images/viewprofile.svg';
import SvgImage from '../../components/SvgImage/SvgImage';
import Colors from '../../theme/colors';

export default function SampleTest(props) {
  const { navigation } = props;
  return (
    <View style={ styles.container }>
      <Text>Welcome to Uprise</Text>

      { /* TODO this is for testing purpose we added code here. this will no longer be here */ }
      <SvgImage iconName={ viewProfileImage } height={ 30 } width={ 30 } />
      <Button
        title='Start'
        loading={ false }
        loadingProps={ { size: 'small', color: Colors.Black } }
        buttonStyle={ {
          backgroundColor: 'rgba(111, 202, 186, 1)',
          borderRadius: 5,
        } }
        titleStyle={ { fontWeight: 'bold' } }
        containerStyle={ {
          width: 100,
          marginVertical: 50,
        } }
        onPress={ () => navigation.navigate('Signup') }
      />
    </View>
  );
}
