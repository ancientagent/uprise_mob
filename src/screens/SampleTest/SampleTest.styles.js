import { StyleSheet } from 'react-native';
import Colors from '../../theme/colors';

export default StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.White,
    alignItems: 'center',
    justifyContent: 'center',
  },
});
