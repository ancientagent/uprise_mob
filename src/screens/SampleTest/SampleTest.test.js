
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SampleTest from './SampleTest';

test('Renders snapshot as expected', () => {
  // const tree = renderer.create(<SampleTest />).toJSON();
  // expect(tree).toMatchSnapshot();
  const renderer = new ShallowRenderer();
  renderer.render(<SampleTest />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
