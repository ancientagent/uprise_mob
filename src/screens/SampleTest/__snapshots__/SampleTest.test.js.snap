// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders snapshot as expected 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Welcome to Uprise
  </Text>
  <Styled(SvgImage)
    iconName={
      Object {
        "testUri": "../../../assets/images/viewprofile.svg",
      }
    }
    size={30}
  />
</View>
`;
